GetUserServices/handler.ts

```diff
@@ -0,0 +1,126 @@
+import * as express from "express";
+
+import {
+  ClientIp,
+  ClientIpMiddleware
+} from "io-functions-commons/dist/src/utils/middlewares/client_ip_middleware";
+
+import {
+  AzureApiAuthMiddleware,
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import {
+  AzureUserAttributesMiddleware,
+  IAzureUserAttributes
+} from "io-functions-commons/dist/src/utils/middlewares/azure_user_attributes";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import {
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { EmailString, NonEmptyString } from "italia-ts-commons/lib/strings";
+
+import {
+  checkSourceIpForHandler,
+  clientIPAndCidrTuple as ipTuple
+} from "io-functions-commons/dist/src/utils/source_ip_check";
+
+import { Context } from "@azure/functions";
+import { identity } from "fp-ts/lib/function";
+import {
+  fromLeft,
+  taskEither,
+  TaskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import { ServiceId } from "io-functions-commons/dist/generated/definitions/ServiceId";
+import { ServiceModel } from "io-functions-commons/dist/src/models/service";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { UserInfo } from "../generated/api-admin/UserInfo";
+import { ServiceIdCollection } from "../generated/definitions/ServiceIdCollection";
+import { APIClient } from "../utils/clients/admin";
+import { ErrorResponses, toErrorServerResponse } from "../utils/responses";
+
+/**
+ * Type of a GetUserServices handler.
+ *
+ * GetUserServices returns a list of ServiceId as output or unauthorized or too many requests
```

Comment:
I think is enought to describe the good path.

Reply:
Fixed in [e040495](https://github.com/pagopa/io-functions-services/pull/68/commits/e040495a4536f99dc6b9b4132552e8959ad1dae0)